doc/main/reference/parallel_phase_for_task_arena.rst

```diff
@@ -0,0 +1,151 @@
+.. _parallel_phase_for_task_arena:
+
+``parallel_phase`` Interface for Task Arena
+====================================================================
+
+.. note::
+    To enable this feature, set ``TBB_PREVIEW_PARALLEL_PHASE`` macro to 1.
+
+.. contents::
+    :local:
+    :depth: 1
+
+Description
+***********
+
+This feature extends the `tbb::task_arena specification <https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/task_scheduler/task_arena/task_arena_cls>`_
+with the following API:
+
+* Adds the ``leave_policy`` enumeration class to ``task_arena``.
+* Adds ``leave_policy`` as the last parameter in ``task_arena`` constructors and ``task_arena::initialize`` methods.
+  This allows you to inform the scheduler about the preferred policy for worker threads
+  when they are about to leave `task_arena` due to a lack of available work.
+* Adds new ``start_parallel_phase`` and ``end_parallel_phase`` interfaces to the ``task_arena`` class
+  and the ``this_task_arena`` namespace. These interfaces work as hints to the scheduler to mark the start and end
+  of parallel work in the arena, enabling different worker thread retention policies.
+* Adds the RAII class ``scoped_parallel_phase`` to ``task_arena``.
+
+API
+***
+
+Header
+------
+
+.. code:: cpp
+
+    #define TBB_PREVIEW_PARALLEL_PHASE 1
+    #include <oneapi/tbb/task_arena.h>
+
+Synopsis
+--------
+
+.. code:: cpp
+
+    namespace oneapi {
+        namespace tbb {
+
+            class task_arena {
+            public:
+
+                enum class leave_policy : /* unspecified type */ {
+                    automatic = /* unspecifed */,
+                    fast = /* unspecifed */,
+                };
+
+                task_arena(int max_concurrency = automatic, unsigned reserved_for_masters = 1,
```

Comment:
What about the copy constructor? It copies settings from the other arena, including the leave policy. I think this behavior should be stated somewhere?

Reply:
Since the API for the copy constructor is not changed, I think the only way to state that is to copy description from the spec and add info about the leave policy.

Reply:
Listed copy constructor in member functions.